src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/table/table.component.spec.ts

```diff
@@ -80,7 +80,7 @@ describe('TableComponent', () => {
   });
 
   it('should prevent propagation of mouseenter event', (done) => {
-    fixture.detectChanges();
+    component.ngOnInit();
```

Comment:
is this a fix for the random unit test failure?

Reply:
Yes, but I wasn't sure till today that it happen to other code changes as well. So I will put that commit into a new pr that fixes the failing test.